views/question-table.pug

```diff
@@ -11,16 +11,20 @@ block styles
         h5.card-title Manage Questions
         h6 Add, modify or delete questions.
       .card-action
-        table.bordered
+        table.bordered.centered
           thead
             tr
               th Visiblity
+                br
+                .chip(id='allVisibility', onClick='updateAllVisibility(true)') Show All
```

Comment:
2 things here with the same id

Reply:
Also use the pug builitin `#allVisibility` instead of shoving it in the brackets.

Reply:
fixed